src/main/java/teammates/ui/controller/AdminEmailComposeSendAction.java

```diff
@@ -98,6 +98,15 @@ protected ActionResult execute() {
                                                         new Text(emailContent),
                                                         null);
             data.emailToEdit.emailId = emailId;
+        } else {
+            if (addressModeOn) {
+                statusToAdmin = "Email sent to " + addressReceiverListString;
+                statusToUser.add(new StatusMessage("Email sent to "
+                        + addressReceiverListString, StatusMessageColor.SUCCESS));
+            } else {
+                statusToAdmin = "Email sent to " + groupReceiverListFileKey;
+                statusToUser.add(new StatusMessage("Email sent to " + groupReceiverListFileKey, StatusMessageColor.SUCCESS));
+            }
```

Comment:
@damithc pretty sure one of each mode can be on, or both at the same time. Will get confirmation from someone more familiar with this system.

@Mynk96 will check on whether both can be on at the same time. If they can't be, there is a logic error now, since that else block gets executed regardless of whether group mode is on or not.